common/djangoapps/third_party_auth/models.py

```diff
@@ -420,10 +417,13 @@ class SAMLProviderConfig(ProviderConfig):
         verbose_name="Advanced settings", blank=True,
         help_text=(
             'For advanced use cases, enter a JSON object with addtional configuration. '
-            'The tpa-saml backend supports only {"requiredEntitlements": ["urn:..."]} '
-            'which can be used to require the presence of a specific eduPersonEntitlement. '
-            'Custom provider types, as selected in the "Identity Provider Type" field, may make '
-            'use of the information stored in this field for configuration.'
+            'The tpa-saml backend supports {"requiredEntitlements": ["urn:..."]}, '
+            'which can be used to require the presence of a specific eduPersonEntitlement, '
+            'and {"extra_field_definitions": [{"name": "...", "urn": "..."},...]}, which can be '
```

Comment:
Now that I've started using this feature in production for one of our enterprise customers, I agree that @bradenmacdonald's comment here would be nice to have.  Also sometimes the URN value is not provided in the `<saml:Attribute>` element  -- for example, in my case the "Name" attribute is "countryCode":  

```
<saml:Attribute Name="countryCode" NameFormat="urn:oasis:names:tc:SAML:2.0:attrname-format:basic">
    <saml:AttributeValue xsi:type="xs:string">in</saml:AttributeValue>
</saml:Attribute>
```

I'm assuming that it's fine to specify "countryCode" for the "urn" value in this case, but we might consider another iteration on this at some point.